repository-data/application/src/main/resources/hcm-config/namespaces/hee/person.yaml

```diff
@@ -166,3 +218,55 @@ definitions:
             wicket.id: ${cluster.id}.field
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
+          /organisation:
+            jcr:primaryType: frontend:plugin
+            caption: Organisation
+            field: organisation
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+          /pronouns:
+            jcr:primaryType: frontend:plugin
+            caption: Pronouns
+            field: pronouns
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+              selectable.options: she/her,he/him, they/them
+          /title:
```

Comment:
Should the title field come before name?

Reply:
It would seem more logical to move Title and Pronouns to the top of the form; maybe below Name though ?